transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/TransformationEditor.java

```diff
@@ -514,4 +530,68 @@ void updateHelpText() {
+                                                                    "    <excludeDependencies>false</excludeDependencies>" + //$NON-NLS-1$
+                                                                    "    <archive>" + //$NON-NLS-1$
+                                                                    "        <manifestEntries>" + //$NON-NLS-1$
+                                                                    "            <Project-Group-Id>${project.groupId}</Project-Group-Id>" + //$NON-NLS-1$
+                                                                    "            <Project-Artifact-Id>${project.artifactId}</Project-Artifact-Id>" + //$NON-NLS-1$
+                                                                    "            <Project-Version>${project.version}</Project-Version>" + //$NON-NLS-1$
+                                                                    "        </manifestEntries>" + //$NON-NLS-1$
+                                                                    "    </archive>" + //$NON-NLS-1$
+                                                                    "</configuration>").getBytes()), //$NON-NLS-1$
+                                          StandardCharsets.UTF_8.name());
+            plugin.setConfiguration(config);
+        } else {
+        	
+        }
+        Xpp3Dom instructions = config.getChild("instructions"); //$NON-NLS-1$
+        if (instructions == null) {
+            instructions = Xpp3DomBuilder.build(new ByteArrayInputStream(("<instructions>" + //$NON-NLS-1$
+                                                                          "</instructions>").getBytes()), //$NON-NLS-1$
+                                                StandardCharsets.UTF_8.name());
+            config.addChild(instructions);
+        }
+        Xpp3Dom importPkg = instructions.getChild("Import-Package"); //$NON-NLS-1$
+        if (importPkg == null) {
+            importPkg = Xpp3DomBuilder.build(new ByteArrayInputStream(("<Import-Package>" + //$NON-NLS-1$
+                                                                       "</Import-Package>").getBytes()), //$NON-NLS-1$
+                                             StandardCharsets.UTF_8.name());
+            instructions.addChild(importPkg);
+        }
+        String importPkgs = importPkg.getValue().trim();
+        if (!importPkgs.contains("com.sun.el;version=")) { //$NON-NLS-1$
+            if (!importPkgs.isEmpty()){
+            	importPkgs += ",\n"; //$NON-NLS-1$
+            }
+            importPkgs += "*,com.sun.el;version=\"[2,3)\";resolution:=optional"; //$NON-NLS-1$
+            importPkg.setValue(importPkgs);
+            try (OutputStream out = new BufferedOutputStream(new FileOutputStream(pomFile))) {
+                MavenPlugin.getMaven().writeModel(pomModel, out);
+                project.getFile("pom.xml").refreshLocal(IResource.DEPTH_ZERO, monitor);
```

Comment:
we could use IMavenConstants.POM_FILE_NAME instead of hardcoding the string pom.xml